demo_sinatra/app.rb

```diff
@@ -61,6 +62,38 @@
   erb :formatters, :layout => :googlecharts_layout
 end
 
+def front_page
+  @links = Daru::DataFrame.new({
+    Examples:  [
```

Comment:
To have neat code, we should write these kinds long of list somewhere else and use it here. That would be easy to understand about the dataframe  parameter.